Symfony/CS/Fixer/Contrib/ProtectedToPrivateFixer.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Filippo Tessarotto <zoeslam@gmail.com>
+ */
+class ProtectedToPrivateFixer extends AbstractFixer
```

Comment:
please make the class `final`, we do this on the 2.x/master line for all fixers and also for new fixers on 1.x